CMakeLists.txt

```diff
@@ -85,6 +85,10 @@ IF(EXISTS "${Trilinos_INCLUDE_DIRS}/KokkosCore_config.h")
   STRING(REGEX MATCH "#define KOKKOS_ENABLE_CUDA" ALBANY_ENABLE_CUDA ${KOKKOS_CORE_CONFIG_FILE})
   IF(ALBANY_ENABLE_CUDA)
     MESSAGE("-- Kokkos is configured to use CUDA, Albany will also.")
+    STRING(REGEX MATCH "#define KOKKOS_ENABLE_CUDA_UVM" ALBANY_ENABLE_CUDA_UVM ${KOKKOS_CORE_CONFIG_FILE})
+    IF(ALBANY_ENABLE_CUDA_UVM)
+      MESSAGE("-- Kokkos is configured to use CUDA UVM, Albany will also.")
+    ENDIF()
     STRING(REGEX MATCH "#define KOKKOS_COMPILER_CUDA_VERSION ([0-9]*)" _ ${KOKKOS_CORE_CONFIG_FILE})
```

Comment:
I know it's not part of your PR, but since we're dealing with cuda...what is the purpose of this line? It seems pointless to me. If it is indeed pointless, can you remove it?

Reply:
I thinkt it's to print the cuda version used to build at runtime. similar to the other compiler prints at runtime.

Reply:
What does it mean for the output variable to be `_` here? I don't think this line even does anything.

Reply:
`_` is a legit CMake variable name. I suppose it's used to signal that this is just a placeholder. However, I'm not sure what's the point of the `STRING` call then, since the output var is the only thing that matter here...

I think we can prune this line.